packages/contracts/src/contracts/ProcessAttestationsVerifier.ts

```diff
@@ -0,0 +1,118 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
We should gitignore typechain files. They should be automatically generated at build time instead.